packages/components/src/__future__/Select/Select.tsx

```diff
@@ -116,13 +116,23 @@ export const Select = <Option extends SelectOption = SelectOption>({
 
   const {
     labelProps,
-    triggerProps,
+    triggerProps: reactAriaTriggerProps,
     valueProps,
     menuProps,
     errorMessageProps,
     descriptionProps,
   } = useSelect(ariaSelectProps, state, triggerRef)
 
+  // Hack incoming:
+  // react-aria/useSelect wants to prefix the combobox's accessible name with the value of the select.
+  // We use role=combobox, meaning screen readers will read the value.
+  // So we're modifying the `aria-labelledby` property to remove the value element id.
+  // Issue: https://github.com/adobe/react-spectrum/issues/4091
+  const triggerProps = {
+    ...reactAriaTriggerProps,
+    "aria-labelledby": reactAriaTriggerProps["aria-labelledby"]?.split(" ")[1],
```

Comment:
If there's no item selected, then would you end up with no label?

Reply:
They still add both ids even when there's no value or placeholder.

<img width="1825" alt="Screenshot 2023-11-21 at 10 17 22 am" src="https://github.com/cultureamp/kaizen-design-system/assets/1811583/dc614a5e-f6b0-4ebc-8f54-1fb6196b39d8">

Reply:
I think there was some weird behavior around it add the default value in if no item was selected. That said, if you have neither that's still a very good question

Reply:
I'll add a test for the case where it doesn't have a selected value if it doesn't already exist

Reply:
Do we pass in the label as a prop? What if we have `aria-label={label}` and `aria-labelledby={undefined}` (to remove what react-aria puts in)?

Reply:
I don't think it's possible to do that. If you pass in `aria-labelledby={undefined|null|" "}` react-aria just ignores it and does its usual thing.

Passing in an extra `aria-labelledby` was possible though, so I've adjusted the string manipulation now to take everything beyond the first space character and added a test for that case.

Reply:
It should only ignore it if you've declared your prop before the spread. If you put it after, it shouldn't be ignored 🤔 Nonetheless, whatever works! No idea what strange magic they may have built in haha.